### PR TITLE
Novel mutations bug fix, recurrent output fix

### LINKED ISSUE #81: [BUG] Error in `novel_mutations.py` when there are no new mutations, error with null `gff_feature`
**Describe the bug**
There is an error in the function `add_features_new_mutations` when there are no new mutations detected. The lambda functions expect values returned when there are none.

**Solution**
If the dataframe is empty, return early.

**Describe the bug**
There is an error in the function `add_features_new_mutations` when there is a mutation has a null `gff_feature`. When calling the lambda function in line 413, the item returned by the `if else` statement for a null `gff_feature` is a list of 5 while the item returned for a non-null `gff_feature` is a `pd.Series` with a length of 5. The shapes are transposed, leading to a `ValueError`.
Note- I'm not sure why this is an error now. This situation has occurred many times before without an error (this bug occurred for position 29804). Nothing has changed about the docker container or any software versions.

**Solution**
Have the `else` part return an empty pd.Series of length 5.

### PR DESCRIPTION
**Pull request by arianna-smith**
This PR 
- closes #81 

## Aim, context, and functionality 🎯
Changes:
- fix a bug in `novel_mutations.py` that occurs when there are no new mutations seen
- only output a recurrent mutations file if there are recurrent mutations seen
- ignore pandas FutureWarnings

### Upstream Effects
None.

### Input Changes
None.

### Output Changes
None.

### Downstream Effects
None.

## Testing 🛠️

**Test dataset(s):**
covwwt_0371_nextseq

**Test(s) performed:**
Ran script to make sure files were produced properly.

**Results (including if the results matched the expected results):**
Results were as expected.

## Developer Checklist 👷‍♀️

- [x] Code has been refactored to sufficiently address the issues this pull request closes
- [x] Testing was performed and the results from testing match the expected results
- [x] All code changes match our style guide
- ~~ Prior to development, issues were discussed with the bioinformatics team members and approved~~ high priority bug fix
- ~~README has been updated to reflect changes~~ n/a
- ~~Workflow diagrams in READMEs have been updated to reflect changes~~

## Reviewer Checklist 🔍

- [x] Met with developer to review all changes and testing performed
- [x] Code refactoring sufficiently address the issue(s) that this pull request closes
- [x] All code meets style guide critera
- [x] Confirm testing was sufficient (e.g. correct dataset was used for testing, results match the expected results). If not, the developer will perform additional testing which should be documented in the testing section above.
- [x] New version release number has been decided upon (if applicable) #patch

### CODE REVIEW

File: scripts/novel_mutations.py [hunk 1]
```diff
@@ -9,7 +9,9 @@
 import argparse
 import sys
 import math
+import warnings
 
+warnings.simplefilter(action='ignore', category=FutureWarning)
```
Comment (danpolanco) [inline scripts/novel_mutations.py:14]: We may want to document which warning we are ignoring somewhere. Could be a comment in the code or maybe just attached to this issue.

**Review by danpolanco — APPROVED** [review]
Looks good to me!

File: scripts/novel_mutations.py (hunk 1 above)
Comment (arianna-smith) [reply scripts/novel_mutations.py:14]: The warning is for concatenating with empty dataframes :)

**Comment by arianna-smith** [comment]
Reopening for another bug found

**Pull request merged**